### ISSUE TITLE
106.11.0 Nuget Upgrade Exception

### ISSUE THREAD

**Issue by JoJo2406** [restsharp/RestSharp#1459]
I am trying to update from 106.10.1 to 106.11.0 however I'm getting the following exception due to the [RestSharp.Serializers.SimpleJson](https://www.nuget.org/packages?q=RestSharp.Serializers.SimpleJson) package not being published to nuget:

```
Attempting to resolve dependencies for package 'RestSharp.106.11.0' with DependencyBehavior 'Lowest'
Unable to resolve dependency 'RestSharp.Serializers.SimpleJson'. Source(s) used: 'NuGet official package source', 'nuget.org', 'Microsoft Visual Studio Offline Packages'.
System.InvalidOperationException: Unable to resolve dependency 'RestSharp.Serializers.SimpleJson'. Source(s) used: 'NuGet official package source', 'nuget.org', 'Microsoft Visual Studio Offline Packages'. ---> NuGet.Resolver.NuGetResolverConstraintException: Unable to resolve dependency 'RestSharp.Serializers.SimpleJson'. Source(s) used: 'NuGet official package source', 'nuget.org', 'Microsoft Visual Studio Offline Packages'.
   at NuGet.Resolver.PackageResolver.Resolve(PackageResolverContext context, CancellationToken token)
   at NuGet.PackageManagement.NuGetPackageManager.<PreviewInstallPackageAsync>d__68.MoveNext()
   --- End of inner exception stack trace ---
   at NuGet.PackageManagement.NuGetPackageManager.<PreviewInstallPackageAsync>d__68.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at JetBrains.ProjectModel.NuGet.Operations.NuGetInstallOperation.<InstallMultiAsync>d__7.MoveNext()
NuGet.Resolver.NuGetResolverConstraintException: Unable to resolve dependency 'RestSharp.Serializers.SimpleJson'. Source(s) used: 'NuGet official package source', 'nuget.org', 'Microsoft Visual Studio Offline Packages'.
   at NuGet.Resolver.PackageResolver.Resolve(PackageResolverContext context, CancellationToken token)
   at NuGet.PackageManagement.NuGetPackageManager.<PreviewInstallPackageAsync>d__68.MoveNext()
```

**Comment by schuettecarsten** [comment]
I can confirm. RestSharp.Serializers.SimpleJson is not (yet) available on Nuget.

**Comment by alexeyzimarev** [comment]
It shouldn't be a separate package. Strange that `dotnet pack` hasn't embedded the dependency. I guess that's because it's not really a dependency. I now have to delist the package, sorry about the issue.

**Comment by pian0** [comment]
Any update on this? I'm getting the same error trying to install 106.11.1

**Comment by jstevenson81** [comment]
I'm having the same problem.

**Comment by alexeyzimarev** [comment]
Ok, 106.11 is unlisted and 106.11.1 will be soon published. I checked the package and it contains both dlls.

**Comment by alexeyzimarev** [comment]
@pian0 maybe you can help? I have no clue, tbh.

The 106.11.1 contains both dlls

![image](https://user-images.githubusercontent.com/2821205/81342996-7d5d4900-90b4-11ea-92fb-fcbe08cbd306.png)

**Comment by alexeyzimarev** [comment]
Ok, I moved SimpleJson back to the core library and published 106.11.2. Other 106.11 versions are unlisted.
